dotcom-rendering/src/lib/assets.ts

```diff
@@ -47,31 +37,63 @@ const isDev = process.env.NODE_ENV === 'development';
 
 export const ASSET_ORIGIN = decideAssetOrigin(process.env.GU_STAGE, isDev);
 
-export const getScriptArrayFromFile = (
+const getManifest = (path: string): AssetHash => {
+	try {
+		const assetHash: unknown = JSON.parse(
+			readFileSync(resolve(__dirname, path), { encoding: 'utf-8' }),
```

Comment:
Does this file get re-read for every request? 

I think previously this file being would be read at app startup, and then would exist in-memory from that point, I'd be interested whether re-reading it would have any impact on rendering latency?

Reply:
I guess that is a valid point, I do not have a strong feeling about this, but feels like this could be revisited at a later point if we ever see any actual performance impact. I would hope reading files from disk is not our bottleneck.

The easiest thing to recreate the existing behaviour would be to create an object one with the mappings:

```ts 
const manifests: { [key: string]: AssetHash } = {
  './manifest.modern.json': getManifest('./manifest.modern.json'),
  './manifest.legacy.json': getManifest('./manifest.legacy.json'),
  './manifest.variant.json': getManifest('./manifest.variant.json'),
} ;
```